### PR TITLE
AWS: Added more DynamoDB and IAM resources

### PR DESCRIPTION
**Pull request by meshuga**
Sorry for number of rows in autogenerated files being reordered. I believe it's the correct (alphaumeric) order.

### CODE REVIEW

File: diagrams/aws/security.py [hunk 1]
```diff
@@ -40,6 +44,22 @@ class Guardduty(_Security):
     _icon = "guardduty.png"
 
 
+class IdentityAndAccessManagementIamAccessAnalyzer(_Security):
```
Comment (mingrammer) [inline diagrams/aws/security.py:47]: Please alias for this: `IAMAccessAnalyzer`

File: diagrams/aws/security.py [hunk 2]
```diff
@@ -40,6 +44,22 @@ class Guardduty(_Security):
...
+    _icon = "identity-and-access-management-iam-access-analyzer.png"
+
+
+class IdentityAndAccessManagementIamAWSSts(_Security):
```
Comment (mingrammer) [inline diagrams/aws/security.py:51]: Please alias for this: `IAMAWSSts`

File: diagrams/aws/security.py [hunk 3]
```diff
@@ -40,6 +44,22 @@ class Guardduty(_Security):
...
+    _icon = "identity-and-access-management-iam-aws-sts.png"
+
+
+class IdentityAndAccessManagementIamPermissions(_Security):
```
Comment (mingrammer) [inline diagrams/aws/security.py:55]: Please alias for this: `IAMPermissions`

File: diagrams/aws/security.py [hunk 4]
```diff
@@ -40,6 +44,22 @@ class Guardduty(_Security):
...
+    _icon = "identity-and-access-management-iam-permissions.png"
+
+
+class IdentityAndAccessManagementIamRole(_Security):
```
Comment (mingrammer) [inline diagrams/aws/security.py:59]: Please alias for this: `IAMRole`

File: diagrams/aws/database.py [hunk 5]
```diff
@@ -24,6 +24,18 @@ class DocumentdbMongodbCompatibility(_Database):
     _icon = "documentdb-mongodb-compatibility.png"
 
 
+class DynamodbDax(_Database):
+    _icon = "dynamodb-dax.png"
+
+
+class DynamodbGlobalSecondaryIndex(_Database):
```
Comment (mingrammer) [inline diagrams/aws/database.py:31]: Please alias for this: `DynamodbGIS`

File: diagrams/aws/database.py [hunk 6]
```diff
@@ -24,6 +24,18 @@ class DocumentdbMongodbCompatibility(_Database):
     _icon = "documentdb-mongodb-compatibility.png"
 
 
+class DynamodbDax(_Database):
```
Comment (mingrammer) [inline diagrams/aws/database.py:27]: Please alias for this: `DAX`

File: diagrams/aws/security.py (hunk 3 above)
Comment (meshuga) [reply diagrams/aws/security.py:55]: fixed

**Pull request merged**